_includes/navigation.html

```diff
@@ -15,50 +15,56 @@ <h2 class="pldoc-heading pldoc-nav-title hd-6">Elements</h2>
     </ul>
 </nav>
 
-<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation">
-    <h2 class="pldoc-heading pldoc-nav-title hd-6">Documentation</h2>
-    <ul class="pldoc-list nav-list list-styleguides">
+<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation" role="navigation" aria-label="Documentation">
+    <h2 class="pldoc-heading pldoc-nav-title hd-6" id="heading-documentation">Documentation</h2>
+    <ul class="pldoc-list nav-list list-styleguides" aria-labelledby="heading-documentation">
         <li class="nav-list-item">
             <a class="pldoc-link" rel="external" href="https://github.com/edx/ux-pattern-library/wiki/Styleguide:-General">
                 <span class="pldoc-icon" aria-hidden="true">&#65514;</span>
                 General Styleguide
+                <span class="sr-only">Link opens in a new window</span>
```

Comment:
@talbs Why the parens? This text isn't seen, only announced, and the goal here is to make it read comfortably. "Link, General Styleguide, Link opens in a new window". I'm willing to simplify to just "opens in a new window", but I'd rather not complicate how it's ready with parens (sometimes they're announced, too). Thoughts?